lib/full_text_search/markup_parser.rb

```diff
@@ -16,8 +16,10 @@ def parse(object, attribute, options={})
         textilizable(object, attribute, options)
       end
       document = Document.new
-      parser = Nokogiri::HTML::SAX::Parser.new(document)
-      parser.parse(html)
+      if html.presence
+        parser = Nokogiri::HTML::SAX::Parser.new(document)
+        parser.parse(html)
+      end
```

Comment:
Can we use an early return instead?

```ruby
def parse
  html = ...
  return ["", []] unless html.present?
  document = Document.new
  ...
end
```

Reply:
fix: [1a0fc37](https://github.com/clear-code/redmine_full_text_search/pull/161/commits/1a0fc3747c672591ec6d4a2e0be9a6fd927bc66a) Sure!